### PR TITLE
add charakterziffer

### PR DESCRIPTION
**Pull request by charakterziffer**

### CODE REVIEW

File: 2024.html
Comment (j9t) [inline 2024.html:65]: Please move one up (sort by alphabet).

(Your call, but please also check on case and if you want to disclose your name, in case of a personal website.)

**Review by j9t — CHANGES_REQUESTED** [review]

File: 2024.html
Comment (arkhi) [reply 2024.html:65]: @j9t: Thanks for al the work.

Since we’re sorting by alphabetically order, “charakterziffer” comes after “Burnell”, so I’ll merge this. We can still change things if need be.

**Review by arkhi — APPROVED** [review]

File: 2024.html
Comment (j9t) [reply 2024.html:65]: Strange, not sure what happened here. Must have missed something.

Also not sure about the note to sort alphabetically, @arkhi, if that’s precisely what I asked to do.

**Pull request merged**
